### PR TITLE
Bunnyman distortion and distortion adjustments

### PR DESCRIPTION
**Pull request by Coxswain-Navigator**
## About The Pull Request
Adds the Bunnyman, the first urban plague distortion. Also makes some adjustments to the mob type to allow for higher risk level armors and fixes a runtime related to mob AI.

## Why It's Good For The Game
It's awesome.

## Changelog
:cl:
add: Added the bunnyman distortion
/:cl:

### CODE REVIEW

**Comment by Gboster-0** [comment]
appears to break norinori scream and breach icons

**Comment by Coxswain-Navigator** [comment]
Might be an icon bot problem because I didn't touch them, but i'll double check.

File: code/modules/mob/living/simple_animal/distortion/plague/bunnyman.dm [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+	egoist_attributes = 80
+	//Loot on death; distortions should be valuable targets in general.
+	loot = list(/obj/item/documents/ncorporation, /obj/item/documents/ncorporation) //Placeholder, we need more loot items
+	//Custon unmanifest effect
+	unmanifest_effect = /obj/effect/gibspawner/human
+
+	var/can_act = TRUE
+	var/transformed = FALSE
+
+//Proc that can be used for additional effects on unmanifest
```
Comment (LanceSmites328) [inline code/modules/mob/living/simple_animal/distortion/plague/bunnyman.dm:48]: Consider:
```suggestion
/mob/living/simple_animal/hostile/distortion/bunnyman/ComponentInitialize()
	. = ..()
	AddElement(/datum/element/waddling)

//Proc that can be used for additional effects on unmanifest
```

**Review by Kitsunemitsu — APPROVED** [review]
Looks good to me.

**Pull request merged**